client/webserver/api.go

```diff
@@ -418,6 +419,33 @@ func (s *WebServer) apiUpdateCert(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, simpleAck(), s.indent)
 }
 
+// apiRestoreWalletInfo is the handler for the '/restorewalletinfo' API
+// request.
+func (s *WebServer) apiRestoreWalletInfo(w http.ResponseWriter, r *http.Request) {
+	form := &struct {
+		AssetID uint32
+		Pass    encode.PassBytes
+	}{}
```

Comment:
Can you do a `defer form.Pass.Clear()` after this?

Unrelated, but in the function below I see a `defer form.Pass.Clear()` and a `zero(form.Pass)` which is redundant. I wonder if I did that.

Reply:
I'll remove the `zero(form.Pass)`.